server/src/main/java/io/unitycatalog/server/persist/SchemaRepository.java

```diff
@@ -208,10 +210,19 @@ public SchemaInfo updateSchema(String fullName, UpdateSchema updateSchema) {
         if (updateSchema.getNewName() != null) {
           schemaInfo.setName(updateSchema.getNewName());
         }
+        if (!updateSchema.getProperties().isEmpty()) {
```

Comment:
Please use both `null` and `isEmpty()` checks for properties' absence for consistency.